fpmsyncd/routesync.cpp

```diff
@@ -100,8 +599,14 @@ void RouteSync::onRouteMsg(int nlmsg_type, struct nl_object *obj, char *vrf)
 
     dip = rtnl_route_get_dst(route_obj);
     nl_addr2str(dip, destipprefix + strlen(destipprefix), MAX_ADDR_SIZE);
+#if 0    
```

Comment:
Looks like with this, the VS test is passing, if so, can you please remove this chunk?

Reply:
@prsunny Yes, that is correct this is causing the VS test to fail, Will test it once locally for the side effect of removing this and update by EoD